src/style.css

```diff
@@ -1,3 +1,76 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+html {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
Consider providing a fallback font that is more similar to 'Roboto' than 'serif', such as 'Arial' or 'Helvetica', for better consistency across different browsers and systems.